src/features/filter.ts

```diff
@@ -8,5 +8,16 @@
 export const filterSlice = createSlice({
   name: 'filter',
   initialState,
-  reducers: {},
+  reducers: {
+    setStatus: ({ query }, { payload }: PayloadAction<string>) => ({
+      query,
+      status: payload,
+    }),
+    setQuery: ({ status }, { payload }: PayloadAction<string>) => ({
+      status,
+      query: payload,
```

Comment:
The setQuery reducer function destructures 'status' from the state but does not include 'query'. This might lead to unintended behavior if 'query' is needed elsewhere in the state.